packages/orchestrator-ui-components/src/components/WFOForms/AutoFieldLoader.tsx

```diff
@@ -0,0 +1,60 @@
+import {
+    BoolField,
+    DividerField,
+    LabelField,
+    LongTextField,
+    NumField,
+    SelectField,
+    TextField,
+    ProductField,
+} from './formFields';
+import { Context, GuaranteedProps } from 'uniforms';
+import { AutoField } from 'uniforms-unstyled';
+
+export function autoFieldFunction(
+    props: GuaranteedProps<unknown> & Record<string, unknown>,
+    uniforms: Context<Record<string, unknown>>,
+) {
+    const { allowedValues, fieldType, field } = props;
+    const { format } = field;
+
+    switch (fieldType) {
+        case Number:
+            switch (format) {
+                case 'imsPortId':
```

Comment:
This is a SURF specific field.

Reply:
This is won't fix for the context of this PR